src/olympia/devhub/tests/rss_feeds/blog.xml

```diff
@@ -60,10 +60,10 @@
 			<content:encoded><![CDATA[<p>I post these updates every 2 weeks to inform add-on developers about     the status of the review queues and any other important stuff going  on    in the add-ons world.</p>
 <h3>Summary</h3>
 <ul>
-<li>Nominations for full review at the moment are being processed slower     than usual. The   new   review system required hundreds of add-on     developers to   renominate their   add-ons, which has placed a large     workload on the   editor team. Nominations are currently taking several     weeks (~5) to  review.</li>
+<li>Nominations for full review at the moment are being processed slower     than usual. The   new   review system required hundreds of add-on     developers to   renominate their   add-ons, which has placed a large     workload on the   reviewers team. Nominations are currently taking several     weeks (~5) to  review.</li>
```

Comment:
`reviewers team` or `reviewer team`?  (and elsewhere in this probably entirely obsolete file)

Reply:
yeah it's just a test file so I didn't care much. The only reason I did the replace in that file is to prevent it from polluting my grep results without having to exclude it every time :)